src/app/image-group-page/image-group-page.component.ts

```diff
@@ -309,28 +309,24 @@ export class ImageGroupPage implements OnInit, OnDestroy {
   private exportGroup(exportType: SupportedExportTypes): void {
     this.showTermsConditions = false
 
-    this._ig.getImageIdsToDownload(this.ig)
```

Comment:
Moved the fetching of the `imageIdsToDownload` into the function that handle the loading states for the export. That is how we are able to reintroduce the error handling that used to be in place.